bazelisk.py

```diff
@@ -70,7 +78,10 @@ def find_workspace_root(root=None):
     return find_workspace_root(new_root) if new_root != root else None
 
 
-def resolve_version_label_to_number(bazelisk_directory, version):
+def resolve_version_label_to_number_or_commit(bazelisk_directory, version):
```

Comment:
Same here - from just reading this function, I don't really get what the bool return value is. Maybe just add a short comment what this function returns?